components/sup/src/manager/service/hooks.rs

```diff
@@ -102,6 +102,7 @@ impl Hook for HealthCheckHook {
         let err_path = hooks::stderr_log_path::<Self>(package_name);
         #[cfg(windows)]
         let path = pair.path.to_path_buf();
+        #[allow(clippy::redundant_clone)]
```

Comment:
Why this exception? Is it because of the conditional compilation? If so, a brief note to that effect as a comment would be helpful.

Reply:
Yes, it was due to conditional compilation. I should have just switched the order the variables are used. Fixed in, [4a2616](https://github.com/habitat-sh/habitat/pull/7325/commits/4a26167ab43e0ae9c61f7e2830252a6f345e7747)

Reply:
Nice :+1: